scripts/check-parity-bw-extrinsics-benchmarks-weights.sh

```diff
@@ -1,9 +1,23 @@
 #!/bin/bash
 
 set -e
+
+FILES_WITH_EXTRINSICS=$(grep "pallet::call" pallets/* -R | cut -f1 -d: | sort | uniq)
```

Comment:
Not only calls need benchmarks, hooks like `on_initialize` as well.

Reply:
Will follow up on this in CSUB-389.